### ISSUE TITLE
Issue with displaying nested tabs

### ISSUE THREAD

**Issue by 2lag** [mdn/content#22877]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role

### What specific section or headline is this issue about?

The JavaScript within the "Example" section

### What information was incorrect, unhelpful, or incomplete?

I was using this to create a set of nested tabs, and used the supplied JavaScript, however the nested tabs would not be displayed when the parent tab was selected. I know this is a specific issue, but a second example (or optional addition to current example) could be added to assist anybody else who may run into this like I did.

Here is the additional code that should be added underneath the `// Show the selected panel` section of the existing Javascript:
```
const tabPanels = grandparent.parentNode.querySelectorAll('[role="tabpanel"]');
    tabPanels.forEach(tabPanel => {
      const selectedTabs = tabPanel.querySelectorAll('[aria-selected="true"]');
      selectedTabs.forEach(selectedTab => {
        const selectedTabPanel = document.getElementById(selectedTab.getAttribute('aria-controls'));
        selectedTabPanel.removeAttribute('hidden');
      });
    }
  );
```

### What did you expect to see?

Ideally, any selected tab within the existing tab would automatically display when one of the parent tabs was clicked. This does exactly that, but it only applicable if nested tabs exist. So a check for that will be needed to add this to the existing code.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/accessibility/aria/roles/tab_role`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/roles/tab_role/index.md
* Last commit: https://github.com/mdn/content/commit/7bd1e39d340236003d434d45fcc30a419acc3492
* Document last modified: 2022-11-21T00:36:59.000Z

</details>

**Comment by 2lag** [comment]
Not sure `help wanted` is the proper tag, this is a completed solution. Was just suggesting the addition to those who may be using nested tabs 😄 Thank you though!

**Comment by postmeback** [comment]
Hi @2lag 

https://github.com/mdn/content/pull/27142

if this fixes your issue, requesting you help me with the prettier formatting.

**Comment by 2lag** [comment]
> Hi @2lag 
> 
> https://github.com/mdn/content/pull/27142
> 
> if this fixes your issue, requesting you help me with the prettier formatting. 

It all looks good to me, is there something specific you need help with?

**Comment by postmeback** [comment]
In the PR, I am unable to figure out how the `prettier` error is coming up. If it is possible, can you help on that @2lag

**Comment by 2lag** [comment]
@postmeback Is that not just a yarn package? I just googled "prettier yarn" after looking at the build error and found a link to it.

**Comment by postmeback** [comment]
https://github.com/mdn/content/pull/27142

@2lag , the issue got fixed. Requesting you to recheck

**Comment by 2lag** [comment]
Seems to be good, merge just needs to be accepted as far as I can tell. Once it is, could you let me know and I'll close this issue?
